source/Preferences.cpp

```diff
@@ -104,6 +106,8 @@ void Preferences::Load()
 			screenModeIndex = max<int>(0, min<int>(node.Value(1), SCREEN_MODE_SETTINGS.size() - 1));
 		else if(node.Token(0) == "alert indicator")
 			alertIndicatorIndex = max<int>(0, min<int>(node.Value(1), ALERT_INDICATOR_SETTING.size() - 1));
+		else if(node.Token(0) == "previous saves" && node.Size() >= 2)
+			previousSaveCount = max(node.Value(1), 3.);
```

Comment:
```suggestion
			previousSaveCount = max(node.Value(1), 2.);
```
I don't see why we'd limit it to less than what we support
could we even allow 1.? it is clearly no mistake from the player if they write it, we prob should support it

Reply:
This isn't an index, this value is the number of previous saves that will be made. 3 is the current behaviour.
It now allows any value above 0. Invalid values (0 or less) are set to 3.